@types/gql.d.ts

```diff
@@ -65,17 +67,19 @@ type CreateProjectQueryArgs = {
         tokens: number;
         success: boolean;
         timestamp: number;
+        latency?: number;
     };
 };
 
 type UpdateProjectQueryArgs = {
```

Comment:
To minimize the changes we need to make in the future does it make sense to define a root project query type and then extend the type when we we need to make updates?